v3/testdata/evSanIpAddressPresent.pem

```diff
@@ -0,0 +1,42 @@
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 3 (0x3)
+        Signature Algorithm: ecdsa-with-SHA256
+        Issuer:
+        Validity
+            Not Before: Jan  1 00:00:00 1 GMT
+            Not After : Nov 30 00:00:00 9998 GMT
+        Subject: CN = example.com, O = Example Inc., ST = CA, C = US
+        Subject Public Key Info:
+            Public Key Algorithm: id-ecPublicKey
+                Public-Key: (256 bit)
+                pub:
+                    04:e3:61:a6:16:b1:62:5d:5b:0c:a9:c9:cf:e9:ba:
+                    67:e6:1f:d3:bc:36:b7:ec:5d:5d:61:bb:cc:2e:ab:
+                    2d:13:19:d4:94:70:89:76:a9:61:04:7b:3f:6d:bf:
+                    b8:21:69:12:e1:89:3c:b5:32:ac:ae:e1:86:c8:8e:
+                    c7:8f:fa:8c:3e
+                ASN1 OID: prime256v1
+                NIST CURVE: P-256
+        X509v3 extensions:
+            X509v3 Subject Alternative Name:
+                DNS:example.com, IP Address:1.1.1.1
```

Comment:
I initially decided to not use an IANA-reserved IP address as these must not be included in any certificate, but as the address itself doesn't matter for this test, I agree that this would be better. Updated.